src/mapDataManipulation/effectFunctions.ts

```diff
@@ -66,3 +66,28 @@ export function runColorEffectFunction(
 
   return rgbMappingArrays;
 }
+
+export function runCustomEffectFunctions(
```

Comment:
When playing around with different `customEffect` functions in the storybook example, I noticed that this function gives unexpected results:
```
        customEffect: ({ r, g, b }) => ({
          r: r + g + b < 0.6 ? 0 : 1,
          g: r + g + b < 0.6 ? 0 : 1,
          b: r + g + b < 0.6 ? 0 : 1,
        }),
```

The resulting pixels should be either black or white (unless I'm misunderstanding something), but I get:
![image](https://user-images.githubusercontent.com/40981131/91701669-ebec7800-eb77-11ea-950e-a7b4fdce06d9.png)

Since it is difficult to know if the implementation is correct just by looking at the code, I would suggest writing unit tests for `runCustomEffectFunctions` (and other effect functions, though that might be out of scope in this MR).

Reply:
I wrote some tests, for now just for `runColorEffectFunction` and `runCustomEffectFunction`. I plan to improve gain and gamma in the near future, so I will add them then.